Packs/CommonScripts/Scripts/script-ShowOnMap.yml

```diff
@@ -4,43 +4,65 @@ commonfields:
 name: ShowOnMap
 script: >
   var lat = parseFloat(args.lat);
-
   var lng = parseFloat(args.lng);
+  var address = args.address;
+
+  if (isNaN(lat) || isNaN(lng)) {
+    if (!address || address.trim().length === 0){
+    return {
+        ContentsFormat: formats.text,
+        Type: entryTypes.error,
+        Contents: "Please enter either coordinates (lat,lng) or an address (which requires a GoogleMaps integration instance)."
+    };
 
+    }
+    // An active instance of GoogleMaps exists, geocoding is possible
+      var commandResult = executeCommand("google-maps-geocode", {"search_address" : address});
+      var ec = commandResult[0].EntryContext["GoogleMaps(val.lat && val.lat == obj.lat && val.lng && val.lng == obj.lng)"];
```

Comment:
Is it possible that there will be an entry without `EntryContext`? e.g. an error entry.